attestation-service/verifier/src/az_snp_vtpm/mod.rs

```diff
@@ -43,6 +44,24 @@ impl AzSnpVtpm {
     }
 }
 
+pub(crate) fn extend_claim_with_tpm_quote(
+    claim: &mut TeeEvidenceParsedClaim,
+    quote: &Quote,
+) -> Result<()> {
+    let Value::Object(ref mut map) = claim else {
+        bail!("failed to extend the claim, not an object");
+    };
+
+    let mut tpm_values = serde_json::Map::new();
+    for (i, pcr) in quote.pcrs_sha256().enumerate() {
```

Comment:
where's `pcrs_sha256` coming from? I couldn't track it down.

Reply:
Thanks @mkulke, that info is helpful. Yeah I was sanity-checking the ordering when I asked this question, I believe. LGTM, nice PR.